src/unstract/sdk/utils/tool_utils.py

```diff
@@ -47,16 +53,20 @@ def get_hash_from_file(file_path: str) -> str:
         Returns:
             str: SHA256 hash of the file
         """
-        h = sha256()
-        b = bytearray(128 * 1024)
-        mv = memoryview(b)
-        with open(file_path, "rb", buffering=0) as f:
-            while n := f.readinto(mv):
-                h.update(mv[:n])
-        return str(h.hexdigest())
+
+        # Adding the following DeprecationWarning manually as the package "deprecated"
+        # does not support deprecation on static methods.
+        warnings.warn(
+            "`old_function` is deprecated. Use `new_function` instead.",
```

Comment:
NIT: Please specify the function names correctly - also it'll be good to mention the version it's deprecated from. Check if that's an argument to `warnings.warn()`

Reply:
Thank you for spotting this. Done.